client/src/components/Tool/ToolCard.vue

```diff
@@ -10,26 +10,24 @@
                         title="Options"
                         variant="link"
                         aria-label="View all Options"
-                        class="float-right"
+                        class="tool-dropdown float-right"
                         size="sm"
                         v-b-tooltip.hover
                     >
                         <template v-slot:button-content>
                             <span class="fa fa-caret-down" />
                         </template>
-                        <b-dropdown-item @click="onCopyLink"> <span class="fa fa-chain" /> Copy Link </b-dropdown-item>
-                        <b-dropdown-item @click="onCopyId">
-                            <span class="fa fa-files-o" /> Copy Tool ID
-                        </b-dropdown-item>
-                        <b-dropdown-item v-if="showDownload" @click="onDownload">
-                            <span class="fa fa-download" /> Download
-                        </b-dropdown-item>
-                        <b-dropdown-item v-if="showLink" @click="onLink">
-                            <span class="fa fa-external-link" /> See in Tool Shed
-                        </b-dropdown-item>
-                        <b-dropdown-item v-for="w of webhookDetails" :key="w.title" @click="w.onclick">
-                            <span :class="w.icon" /> {{ w.title }}
-                        </b-dropdown-item>
+                        <b-dropdown-item @click="onCopyLink"><span class="fa fa-chain" />Copy Link</b-dropdown-item>
```

Comment:
If this uses the standard mount, all these labels can be localized with the "L" or "localize" filters or the v-localize directive.

```html
<b-dropdown-item @click="onCopyLink">
<span class="fa fa-chain" />
<span v-localize>Copy Link</spam>
</b-dropdown-item>
```

Reply:
Done.